simulators/taiko/ops/main.go

```diff
@@ -2,57 +2,99 @@ package main
 
 import (
 	"context"
+	"math/big"
 	"time"
 
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/params"
 	"github.com/ethereum/hive/hivesim"
 	"github.com/ethereum/hive/taiko"
 	"github.com/stretchr/testify/require"
+	"github.com/taikoxyz/taiko-client/bindings"
 )
 
-// what we need test of our taiko internal
-// 1. propose 2048 blocks at once
-// 2. propose bad blocks
-// 3. driver sync from zero height
-// 4. driver sync from some none zero height
+var allTests = []*taiko.TestSpec{
+	{Name: "first l2", Description: "generate first verified L2 block", Run: genFirstVerifiedL2Block},
```

Comment:
```suggestion
	{Name: "first L2", Description: "generate first verified L2 block", Run: genFirstVerifiedL2Block},
```
plz use L1 / L2 instead of l1 / l2 in description.